packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -343,7 +343,7 @@ export class Cluster extends Resource implements ICluster {
     const stack = Stack.of(this);
 
     this.vpc = props.vpc || new ec2.Vpc(this, 'DefaultVpc');
-    this.version = props.version;
+    this.version = props.version ?? LATEST_KUBERNETES_VERSION;
```

Comment:
that’s problematic because we can’t have a changing default value (by definition it’s a breaking change). Generally this is a tricky part of this api and I  not sure what’s the best approach to take to be honest. Any ideas?

Reply:
Yes I agree we should not do that. As current default version is 1.14 now and probably will change to 1.15 in the next few weeks. If we need 1.15 during this time, we probably should explicitly specify it.